deploy/common/deploy.sh

```diff
@@ -42,13 +44,43 @@ echo "Image flavor for roxctl set to $ROXCTL_ROX_IMAGE_FLAVOR"
 
 popd
 
-function curl_central() {
-	cmd=(curl --retry 10 --retry-delay 10 --retry-connrefused --silent --show-error --insecure)
-	local admin_user="${ROX_ADMIN_USER:-admin}"
-	if [[ -n "${ROX_ADMIN_PASSWORD:-}" ]]; then
-		cmd+=(-u "${admin_user}:${ROX_ADMIN_PASSWORD}")
-	fi
-	"${cmd[@]}" "$@"
+# curl_central_once
+# Runs curl with --silent --show-error --insecure.
+# Obeys $ROX_ADMIN_USER and $ROX_ADMIN_PASSWORD.
+# arguments:
+#   - anything else `curl` will accept, including the URL
+function curl_central_once() {
+    local cmd=(curl --silent --show-error --insecure)
+    local admin_user="${ROX_ADMIN_USER:-admin}"
+    if [[ -n "${ROX_ADMIN_PASSWORD:-}" ]]; then
+        cmd+=(-u "${admin_user}:${ROX_ADMIN_PASSWORD}")
+    fi
+    "${cmd[@]}" "$@"
+}
+
+
+# curl_central_retry
+# Runs curl like curl_central_once, but with retries and --fail.
+# Captures stdout and emits it only once (for the last invocation of curl).
+# arguments:
+#   - anything else `curl` will accept, including the URL
+function curl_central_retry() {
```

Comment:
It looks great. Do you think we can also use this function under `tests/` dir? There are quite many calls to the Central. There is also similar to `curl_central_once` function called `roxcurl` so it could be replaced.
Nit: provide local name for number of retries or even parametrize it

Reply:
I'd prefer not to extend the scope of this PR further. I took a brief look at some of the other invocations of `curl` on central, and they don't necessarily seem to be covered by CI, so I'm afraid of breaking them.